README.md

```diff
@@ -89,6 +89,9 @@ Feline is a Lua statusline plugin that prioritizes speed, customizability and mi
 **[Config by toufyx](https://gist.github.com/Toufyx/d6b48a0a12ceff02268be49db0a97ddf)**
 ![image](https://user-images.githubusercontent.com/10991276/187600545-034cf707-08c9-456e-af3e-6d4baf1580a9.png)
 
+**[Plugin by Hitesh-Aggarwal](https://github.com/Hitesh-Aggarwal/feline_one_monokai.nvim)**
+![image](https://github.com/Hitesh-Aggarwal/feline_one_monokai.nvim/raw/main/screenshot.png)
```

Comment:
```suggestion
![image](https://raw.githubusercontent.com/Hitesh-Aggarwal/feline_one_monokai.nvim/2ff798d4d0435d2145593587dc93a70e72a6d279/screenshot.png)
```

Makes sense to use a permalink in case you change` screenshot.png`

Reply:
ok